src/hocs/with-active-player/with-active-player.e2e.test.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import {configure, shallow} from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import withActivePlayer from "./with-active-player";
+
+configure({adapter: new Adapter()});
+
+const MockComponent = () => <div />;
+const MockComponentWrapped = withActivePlayer(MockComponent);
+
+it.skip(`Should`, () => {
```

Comment:
Этот тест написан, но с помощью функции `skip` пропущен и не проходит. Надо поправить

```suggestion
it(`Should`, () => {
```